### DISCUSSION TITLE
Does directus schema apply have an auto-yes flag

### DISCUSSION THREAD

**Discussion by iyobo** [directus/directus#9085] [Archive]
I currently have a deployment process where my CI runs `directus schema apply myfile` as a pre-deployment task.

myfile is a snapshot file in my directus repo that I always up to date.
I have noticed that my CI flow hangs on the part where it is expecting you to hit Y for "yes" after it asks you to confirm changes.

Can this be auto set to true so this can work in CI, non-input settings?

**Comment by rijkvanzanten** [comment]
@iyobo try `-y` at the end ✨

**Reply by iyobo** [reply]
@rijkvanzanten thanks!

**Reply by jeremyzilar** [reply]
Thank you!
